appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -456,11 +469,15 @@ public void switchToDesignView() {
     if (currentFileEditor != null) {
       deckPanel.showWidget(designTabIndex);
     } else if (!editorManager.hasOpenEditor()) {  // is there a project editor pending visibility?
-      OdeLog.wlog("No current file editor to show in designer");
+      LOG.warning("No current file editor to show in designer");
       ErrorReporter.reportInfo(MESSAGES.chooseProject());
     }
   }
 
+  public void refreshProperties()
```

Comment:
Since PropertiesBox has a `getInstance` method, why not just use that wherever needed rather than further tightly coupling classes to Ode?